pants.toml

```diff
@@ -16,11 +17,11 @@ root_patterns = [
               "/test_docker"
               ]
 
-[python-setup]
+[python]
```

Comment:
The name of this config is changing: https://www.pantsbuild.org/docs/reference-python There's still some support for `python-setup` but at least one of the keys had moved.